recipes/monocle3-cli/meta.yaml

```diff
@@ -1,30 +1,31 @@
 {% set name = "monocle3-cli" %}
-{% set version = "0.0.7" %}
+{% set version = "0.0.8" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/ebi-gene-expression-group/monocle-scripts/archive/v{{ version }}.tar.gz
-  sha256: 03d4ac8b7b19a8c74d644980f3ae08812aee782112e1e3895ff7be381963cb64
+  sha256: cf59b89f4bab90d88799ac75faedd1104eeaab19833290b7dd8872cf705961b9
 
 build:
-    number: 1
+    number: 0
     noarch: generic
     rpaths:
         - lib/R/lib/
         - lib/
 
 requirements:
     host:
-        - r-base
+        - r-base >=3.6
         - r-optparse
         - r-monocle3 <=0.2.0
```

Comment:
This pin is probably the problematic one. It will also cause trouble very soon as the older package does not get a R4 build. Is that still needed?

Reply:
I think that Ni needed to stay below 0.2.1, but I can check.

Reply:
Although we frequently have issues later down the line when we leave loose our main dependencies, so I think that we do it out of caution for many or our packages.

Reply:
let's remove it for the sake of seeing whether it passes or not.

Reply:
But I would still prefer the pining, we have had issues in a number of packages when we install them later on and they break functionality because of a change in a relevant sub-package.

Reply:
(I think I did try this locally some time ago though...)

Reply:
nope...removing the pinning didn't made any difference (see tests after commit).